api/client/client.ts

```diff
@@ -115,6 +140,7 @@ export class AtatClient implements IAtatClient {
       return cspClientResponse;
```

Comment:
This is a bug on me, but I wanted to mention it here. The deadline should only be added when an `addEnvironment` operation happens, but currently any request that uses the `buildHeaders` method will have the deadline header.